test/test-kitchen.ts

```diff
@@ -95,10 +95,15 @@ test.serial('use as a non-locally installed module', async t => {
   const GTS = `${stagingPath}/kitchen/node_modules/.bin/gts`;
   const tmpDir = tmp.dirSync({keep, unsafeCleanup: true});
   await ncpp('test/fixtures', `${tmpDir.name}/`);
+
   const opts = {cwd: `${tmpDir.name}/kitchen`};
-  // It's important to use `-n` here because we don't want to overwrite
-  // the version of gts installed, as it will trigger the npm install.
-  await simpleExecp(`${GTS} init -n`, opts);
```

Comment:
I changed `-n` to `-y` because a local `gts` must be installed so that the `gts check` below can find the tsconfig-google.json file.